lib/std/debug.zig

```diff
@@ -459,6 +473,12 @@ pub fn dumpStackTraceFromBase(context: *ThreadContext) void {
             // same behaviour for x86-windows-msvc
             const address = if (return_address == 0) return_address else return_address - 1;
             printSourceAtAddress(debug_info, stderr, address, tty_config) catch return;
+
+            depth += 1;
+            if (depth > max_stack_trace_depth) {
+                stderr.print("Abandoned stack trace after {} frames.\n", .{max_stack_trace_depth}) catch {};
```

Comment:
This should probably print an extra line indicating that this is likely a Zig bug.